lib/schedule.js

```diff
@@ -454,7 +478,7 @@ function prepareNextInvocation() {
       job.stopTrackingInvocation(cinv);
 
       job.invoke();
-      job.emit('run');
+      job.emit('run', inv !== null ? inv.fireDate : null, cinv.endDate);
```

Comment:
I want to know that not run next (arguments is null) or the next run time(arguments is inv.fireDate).